logstash-core/lib/logstash/api/commands/stats.rb

```diff
@@ -110,8 +135,25 @@ def plugin_stats(stats, plugin_type)
             end
           end
 
-          def report(stats)
-            {
+          ##
+          # Returns a vertex, decorated with additional metadata if available.
+          # Does not mutate the passed `vertex` object.
+          #
```

Comment:
```suggestion
          #
          # @api private
```

Even though this method must be public to avoid restructuring of the module, the `@api private` comment should remain in order to discourage developers from using it from outside of the module.

Additionally, per my previous comment and by convention, this private-API method definition should go at the _bottom_ of the module definition, below the public-API methods.

> While you may see plenty of evidence to the contrary in this project, by convention in Ruby, we tend to declare methods in an order that indicates publicity instead of oscillating between modes: first all of the public methods, then protected, then private. This allows a reader to quickly understand a module or class's public API quickly, to be aware of the scope of effect of a change to a class's implementation, and to avoid tying one class's behaviour to another class's implementation details.
>
> -- https://github.com/elastic/logstash/pull/10576#discussion_r295419677

Reply:
Thanks, @yaauie. I have replaced the @private marker in the method documentation and moved it to the bottom.